src/EditorFeatures/Core/Implementation/Outlining/AbstractOutliningTaggerProvider.RegionTag.cs

```diff
@@ -23,14 +23,15 @@ protected class RegionTag : IOutliningRegionTag
             private const int MaxPreviewText = 1000;
 
             private readonly AbstractOutliningTaggerProvider<TRegionTag> _provider;
-            private readonly OutliningSpan _outliningSpan;
 
             private readonly ITextBuffer _subjectBuffer;
             private readonly ITrackingSpan _hintSpan;
 
-            public bool IsDefaultCollapsed => _outliningSpan.IsDefaultCollapsed;
-            public bool IsImplementation => _outliningSpan.AutoCollapse;
-            public object CollapsedForm => _outliningSpan.BannerText;
+            public bool IsDefaultCollapsed => OutliningSpan.IsDefaultCollapsed;
+            public bool IsImplementation => OutliningSpan.AutoCollapse;
+            public object CollapsedForm => OutliningSpan.BannerText;
+
+            protected readonly OutliningSpan OutliningSpan;
```

Comment:
Protected in a public type is part of the public surface area, and so changing from field to property is a breaking change.  Now that readonly autoprops are so simple to write, I prefer them to fields for protected data, but it's not a big deal.